client/src/views/client-area/statistics/CustomOrderStatisticsPage.vue

```diff
@@ -336,14 +343,20 @@ const error = ref('');
 onMounted(async () => {
   if (!isDefaultOrganization(props.currentOrganization)) {
     const orgStatsResponse = await BmsAccessInstance.get().getOrganizationStats(props.currentOrganization.bmsId);
+    const organizationStatusResp = await BmsAccessInstance.get().getOrganizationStatus(props.currentOrganization.bmsId);
+    const isBootstrapped = (organizationStatusResp.data as OrganizationStatusResultData).isBootstrapped ?? false;
 
-    if (!orgStatsResponse.isError && orgStatsResponse.data && orgStatsResponse.data.type === DataType.OrganizationStats) {
+    if (!orgStatsResponse.isError && orgStatsResponse.data && orgStatsResponse.data.type === DataType.OrganizationStats && isBootstrapped) {
       organizationStats.value = orgStatsResponse.data;
       totalData.value = organizationStats.value.dataSize + organizationStats.value.metadataSize;
     } else {
       error.value = 'clientArea.statistics.error';
     }
 
+    if (isBootstrapped) {
```

Comment:
Nop.

The logic should be
```
Has a request failed? -> Display a message
Is the org not bootstraped? -> Display a message
Everything good? -> Set the stats, totalData and all that.
```

Early returns would help here, just remember to set `querying.value` to false before returning. I recommend using a try/catch/finally:

```typescript
querying.value = true;
try {
  // !isDefaultOrganization, set the appropriate message, then return
  // Do requests
  // Requests failed, set the appropriate message, then return
  // Org is not bootstrapped, set the appropriate message, then return
  // Set the values
} finally {
  querying.value = false;
}
```
The finally will ensure that `querying` is always false when we exit the function.